src/database/leads/repository.py

```diff
@@ -112,3 +123,40 @@ def fetch_review(self, place_id):
         :return: json contents of desired review
         """
         pass
+
+    def fetch_gpt_result(self, file_id, operation_name):
```

Comment:
for consistency, add @abstractmethod above methods that you intend to be implemented/overwritten :))

Reply:
Addressed.